transcom/mymove
# B 20695 int

## [B-20695](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A983066)

## Summary

Updated 'Full PPM' to 'PPM' on Estimated weights screen for customer, and 'What did you buy?' to 'What did you buy or rent?' on expense entry for customer.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Access the MilMove app
2. Login as a customer
4. Create a new move
5. Create a new PPM
6. Verify the expect screens label says 'PPM' and not 'Full PPM'
7. Add the PPM to the move

8. Log in as a Service Counselor 
9. Approve the move details for the Move you created

10. Log in as the previously used Customer
11. Navigate to the PPM the move you created
12. Select to upload PPM receipts
13. Select to add an expense
14. Select and expense an verify that 'What did you buy or rent?' is displayed instead of 'What did you buy?'

## Screenshots
![Screenshot 2024-07-18 at 9 37 56 AM](https://github.com/user-attachments/assets/66fc46b0-d58b-4b73-a4a1-4167e8270a8f)
![Screenshot 2024-07-18 at 9 36 29 AM](https://github.com/user-attachments/assets/f1dfd9ef-7058-4d24-852d-840d87edc329)
